.github/workflows/llvm-binaries.yml

```
@@ -204,13 +213,15 @@ jobs:
         with:
           llvm_version: ${{ env.llvm_version }}
 
-      - uses: seanmiddleditch/gha-setup-ninja@master
+      - name: Get latest CMake and Ninja
```

```suggestion
      - name: Install CMake and Ninja
```

It's not necessarily the latest, is it?  Did you write this yourself?

---

Thanks, seems I just copied it from the Hylo repo and forgot to change the title.